pkg/query-service/app/parser.go

```diff
@@ -355,6 +355,11 @@ func parseFilteredSpanAggregatesRequest(r *http.Request) (*model.GetFilteredSpan
 		}
 	}
 
+	groupBy := postData.GroupBy
+	if groupBy == "none" {
+		postData.GroupBy = ""
+	}
+
```

Comment:
The general rule of thumb is the convenience changes on the frontend should end in the frontend only because API serves different platforms/clients. I would avoid this kind of parsing and have clear separation concerns b/w frontend and the backend.

Reply:
@srikanthccv when `groupBy` param is set to `none` it should be parsed empty string. This is a common practice to set param as `none`.

Reply:
Where did you see this being common practice? `none`, `nil`, `empty` etc are all frontend play, and they don't get sent as is in the request payload and then parsed on the API side. Take a look at honeycomb/lightstep API for example. Usually, zeroed values are sent to indicate no selection except the zeroed values themselves are possible input, in which case JSON null is used.

Reply:
Made the change to handle this on frontend